README.md

```diff
@@ -6,3 +6,25 @@ Terraform module to provision an [`RDS Aurora`](https://aws.amazon.com/rds/auror
 ## License
 
 Apache 2 License. See [`LICENSE`](LICENSE) for full details.
+
+
+## Usage
+
+Include this repository as a module in your existing terraform code:
+
+```terraform
+module "rds_cluster_aurora_acig" {
```

Comment:
should not use `asig` in the generic module, it's a customer specific name.
`module "rds_cluster_aurora"` should be enough.